pallets/messages/src/benchmarking.rs

```diff
@@ -92,21 +99,5 @@ benchmarks! {
 		}
 	}: _ (RawOrigin::Signed(caller),IPFS_SCHEMA_ID, cid, IPFS_PAYLOAD_LENGTH)
 
-	on_initialize {
```

Comment:
Let's consider adding a final check of the expected end state to the above benchmarks.